.github/workflows/coordinator-test-v21.yml

```diff
@@ -108,7 +108,7 @@ jobs:
       # (see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)
       fail-fast: false
       matrix:
-        backend: [ cassandra-40, cassandra-311, dse-68 ]
+        backend: [ cassandra-40, cassandra-311, dse-68, dse-v4-68 ]
```

Comment:
I know it's just an arbitrary name but what does `v4` here mean? Based on Cassandra v4.x? (would this be dse-69?)

Reply:
Dse run with V4 Protocol.

Reply:
Would have been nice to add a comment somewhere if anyone else was wondering but maybe this is only temporary addition.